reana_cluster/version.py

```diff
@@ -27,4 +27,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "0.3.1"
+__version__ = "0.3.2"
```

Comment:
We need to upgrade the docker tags in the `reana-cluster[|-latest|-dev].yaml`:
https://github.com/reanahub/reana-cluster/blob/master/reana_cluster/configurations/reana-cluster.yaml

[`reanahub/reana-workflow-controller:0.3.2`](https://github.com/reanahub/reana-workflow-controller/commit/ca8383c99d1d411412c9b27176f02fff0beb2fa7)
[`reanahub/reana-workflow-engine-serial:0.3.2`](https://github.com/reanahub/reana-workflow-engine-serial/commit/d8421d5804bd757d5e167b037fd033795e4ba09f)
[`reanahub/reana-workflow-engine-cwl:0.3.2`](https://github.com/reanahub/reana-workflow-engine-cwl/commit/22553c3c4ca437108f4db1153c853823e4e47480)